src/ckeditor.jsx

```diff
@@ -17,18 +17,17 @@ export default class CKEditor extends React.Component {
 		this.domContainer = React.createRef();
 	}
 
-	componentDidUpdate() {
-		if ( !this.editor ) {
-			return;
+	// This component should never be updated by React itself.
+	shouldComponentUpdate( nextProps ) {
+		if ( this._shouldUpdateContent( nextProps ) ) {
+			this.editor.setData( nextProps.data );
```

Comment:
This was not the right place to do this. :|

Reply:
I have editor content saved as a draft in local storage, and it is picked up and sent to CKEditor as a prop in data. Of course the value changes for data, but CKeditor doesn't re-render.

Reply:
Please, report an issue if you think that this part of the code is invalid.